deployment/helm/node-feature-discovery/values.yaml

```
@@ -13,6 +13,7 @@ namespaceOverride: ""
 enableNodeFeatureApi: true
 
 master:
+  enabled: true
```

We have `gc.enable`, `topologyUpdater.enable`, `prometheus.enable` so for consistency it'd be good to have this `master.enable`

```suggestion
  enable: true
```